app/src/main/java/in/avimarine/boatangels/db/FireBase.java

```diff
@@ -77,16 +80,26 @@ public Boat getBoat (UUID uuid){
       return null;
     }
 
+
+
   @Override
   public void getBoatsInMarina(String marina, OnCompleteListener<QuerySnapshot> listener ) {
     mFirestore.collection("boats").whereEqualTo("marinaName", marina).get().addOnCompleteListener(listener);
   }
 
+
+
   @Override
   public void addInspection(Inspection i) {
     mFirestore.collection("inspections").document(i.getUuid()).set(i);
   }
 
+  @Override
+  public void addusers(User user) {
+      mFirestore.collection("Users").document(user.getUuid()).set(user);
```

Comment:
I think you should save the user using it's Uid and not UUID

Reply:
done